test/crud-with-options.test.js

```diff
@@ -536,6 +536,54 @@ describe('crud-with-options', function() {
 
   });
 
+  describe('upsertWithWhere', function() {
+
+    beforeEach(seed);
+
+    it('should allow upsertWithWhere(data, cb)', function(done) {
```

Comment:
As I commented earlier, this does not make sense to me. What's the point of `upsertWithWhere(data, cb)` when there is now `where` argument? That's just regular `upsert`.